nhaystack-rt/src/nhaystack/server/TagManager.java

```diff
@@ -485,7 +485,7 @@ public HDict createComponentTags(BComponent comp)
             // add misc other tags
             hdb.add("axType", comp.getType().toString());
             if (comp.getSlotPath() != null)
-                hdb.add("axSlotPath", comp.getSlotPath().toString());
+                hdb.add("n4SlotPath", comp.getSlotPath().toString());
```

Comment:
We can't just change how we treat the axSlotPath tag.  There will be existing systems that use it needs to be kept.